log/logger.go

```diff
@@ -0,0 +1,97 @@
+// Package log provides sane default loggers using slog.
+package log
+
+import (
+	"io"
+	"log/slog"
+	"os"
+	"strings"
+	"time"
+)
+
+const DefaultRefreshInterval = time.Second * 5
+
+// NewDefaultEnvLogger creates new slog.Logger using sane default configuration and sets it as a default logger.
```

Comment:
Documented function name `NewDefaultEnvLogger` does not match the declared `NewDefaultLogger`. Should rename to `.*Env.*` as it seems to get the log level from environment or adjust the documented function name? What should `NewDefaultLogger` do?

Reply:
Fixed